data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -22,6 +22,7 @@ public class KeyValueProcessorConfig {
     static final String DEFAULT_PREFIX = "";
     static final String DEFAULT_DELETE_KEY_REGEX = "";
     static final String DEFAULT_DELETE_VALUE_REGEX = "";
+    static final String DEFAULT_TRANSFORM_KEY = "";
```

Comment:
Since you already set the default value to be empty string, null check can be removed in `KeyValueProcessor`